ci/backup-metadata/install-cf-for-k8s.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+ROOT_DIR=$PWD
+
+cluster_name=$(cat $ROOT_DIR/cluster-name/info)
+dns_domain="${cluster_name}.${DNS_ROOT_DOMAIN}"
+
+# temporary hack
+wget -O- https://k14s.io/install.sh | bash
```

Comment:
Potential for version mismatches with the CAPI CI images?

Reply:
Yep for sure, I think it makes more sense to integrate into the capi pipeline currently which does this installation instead of it being a separate pipeline. However for the moment until we figure that out, this was a temporary workaround.